test/integration/infra.go

```diff
@@ -308,6 +308,14 @@ func (infra *infra) deployApp(deployment, svcName string, port1, port2, port3, p
 }
 
 func (infra *infra) teardown() {
+
+	if yaml, err := fill("rbac-beta.yaml.tmpl", infra); err != nil {
```

Comment:
why is that still needed now that the rbac is inlined into istio.yaml ?

Reply:
because the e2e pilot processes and applies the templates individually, not the istio.yaml.
I am making a similar change in istio/istio e2e, part of the tests do the same (I know this is not correct, I had opened an issue).